target/linux/ramips/dts/mt7620a_netgear_ex3700.dts

```diff
@@ -2,26 +2,12 @@
 
 /dts-v1/;
 
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
-
-#include "mt7620a.dtsi"
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
 
 / {
 	compatible = "netgear,ex3700", "ralink,mt7620a-soc";
 	model = "Netgear EX3700/EX3800";
 
-	aliases {
-		led-boot = &led_power_green;
-		led-failsafe = &led_power_green;
-		led-running = &led_power_green;
-		led-upgrade = &led_power_green;
-	};
```

Comment:
Keep the aliases together with the corresponding leds definition (i.e. in this file).